Gruntfile.coffee

```diff
@@ -403,18 +411,18 @@ module.exports = (grunt) ->
 				options:
 					banner: ""
 					process: ( src, filepath ) ->
-						lang = filepath.replace grunt.config( "coreDist" ) + "/js/i18n/", ""
+						lang = filepath.replace( grunt.config "coreDist" + "/js/i18n/", "")
 						# jQuery validation uses an underscore for locals
 						lang = lang.replace "_", "-"
 						validationPath = "lib/jquery-validation/src/localization/"
 
 						# Check and append message file
 						messagesPath = validationPath + "messages_" + lang
-						messages = if grunt.file.exists messagesPath then grunt.file.read( messagesPath ) else ""
+						messages = if grunt.file.exists messagesPath then grunt.file.read messagesPath else ""
```

Comment:
More `grunt.` stuff that I guess should be `@.` if that's the pattern you're going to go with